src/back/glsl/features.rs

```diff
@@ -101,7 +103,10 @@ impl FeaturesManager {
         check_feature!(SAMPLE_VARIABLES, 400, 300);
         check_feature!(DYNAMIC_ARRAY_SIZE, 430, 310);
         check_feature!(MULTI_VIEW, 140, 310);
-        check_feature!(FMA, 400, 310);
```

Comment:
I guess this line was accidentally removed ?

Reply:
No this line is correct, `check_feature!` is only used for features that cannot be workarounded, fma can.